### PR TITLE
[new-exec] Refine standalone executor

### PR DESCRIPTION
**Pull request by zhiqiu**
### PR types

Others

### PR changes

Others
### Describe

Refine standalone executor

- add `feed`  ops in python sides to be consistent with old executor and support more feed type

### CODE REVIEW

File: python/paddle/fluid/executor.py [hunk 1]
```diff
@@ -493,29 +493,20 @@ def __init__(self, place, main_program, scope):
...
-                The default is None.
+            feed_names(list): This parameter represents the input names of the model.
             fetch_list(list): This parameter represents the Tensors that need to be returned
                 after the model runs. The default is None. 
             return_numpy(bool): This parameter indicates whether convert the fetched Tensors
                 (the Tensor specified in the fetch list) to numpy.ndarray. if it is False,
                 the type of the return value is a list of :code:`LoDTensor`. The default is True.
         """
-        feed = self._update_feed(feed)
+        # feed = self._update_feed(feed)
```
Comment (Aurelius84) [inline python/paddle/fluid/executor.py]: 注释的代码后续可以考虑remove掉

**Review by Aurelius84 — DISMISSED** [review]

**Review by wanghuancoder — DISMISSED** [review]
LGTM

File: python/paddle/fluid/executor.py (hunk 1 above)
Comment (zhiqiu) [reply python/paddle/fluid/executor.py]: done, thx

**Review by pangyoki — APPROVED** [review]
LGTM

**Pull request merged**